packages/amplify-java-template-provider/resources/lambda/hello-world/build.gradle.ejs

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'java'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile (
+        'com.amazonaws:aws-lambda-java-core:1.2.0',
+        'com.amazonaws:aws-lambda-java-events:2.2.7'
+    )
+}
+
+jar.baseName = 'latest_build'
```

Comment:
If possible, we shouldn't be hardcoding this filename and instead get it from the package request. Maybe this SO post can help: https://stackoverflow.com/questions/38222724/rename-file-in-gradle-with-passed-parameter